alphabase/psm_reader/alphapept_reader.py

```diff
@@ -1,4 +1,4 @@
-import os
+from pathlib import Path
```

Comment:
Is this a thing? that `Path` is prefered over os?

Reply:
yes.. imho it makes path operations much more readable, 
I especially like the overloading of the `/` operator when it comes to using it (e.g. `Path("/home/") / "user"`).
I especially dislike the overloading of the `/` operator when it comes to mocking it :-)